client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -112,9 +117,22 @@ const PluginsBrowserListElement = ( props ) => {
 		return ! isCompatiblePlugin( plugin.slug ) && ! jetpackNonAtomic;
 	} );
 
-	const shouldUpgrade = useSelector( ( state ) => shouldUpgradeCheck( state, selectedSite?.ID ) );
+	const isMarketplaceAddonCompatible = useSelector( ( state ) =>
+		siteIsMarketplaceAddonCompatible( state, selectedSite?.ID )
+	);
+
+	const canInstallPurchasedPlugins =
+		useSelector( ( state ) =>
+			siteHasFeature( state, selectedSite?.ID, WPCOM_FEATURES_INSTALL_PURCHASED_PLUGINS )
+		) && isMarketplaceAddonCompatible;
+
+	const canInstallPlugins =
+		useSelector( ( state ) =>
+			siteHasFeature( state, selectedSite?.ID, WPCOM_FEATURES_INSTALL_PLUGINS )
+		) && isMarketplaceAddonCompatible;
```

Comment:
This is because it effectively checks to make sure the site is not Jetpack (non-Atomic) or VIP. Maybe we should rename `siteIsMarketplaceAddonCompatible` to `siteIsNotStandaloneJetpackOrVip`?

Reply:
Oh, I see. Yeah, that's a better name that would avoid confusion to others 🙂